pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/NamespacesTest.java

```diff
@@ -1249,9 +1250,11 @@ public void testSubscribeRate() throws Exception {
         Awaitility.await().untilAsserted(() -> assertFalse(consumer.isConnected()));
 
         // Out of limit period
-        Thread.sleep(6000L);
         pulsarClient.updateServiceUrl(lookupUrl.toString());
-        assertTrue(consumer.isConnected());
+        Awaitility.await().atLeast(5, TimeUnit.SECONDS)
```

Comment:
there is no need to specify `.atLeast` or `.atMost` parameters. Let's just rely on the default value of 10 seconds for `.atMost` in this case.
`.atLeast` is probably wrong. You might have meant `.pollDelay` which specifies the initial waiting time before making the first check. There's also `.pollInterval` and `.pollDelay` will default to `.pollInterval` unless it's explicitly set.

In general, it's better to omit the settings unless there's a specific reason to set `.pollDelay`, `.pollInterval` or `.atMost`.
One reason could be that the polling consumes a lot of resources or more than 10 seconds should be waited for the result.